clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -13826,6 +13826,9 @@ void NamespaceRule::runRule(const MatchFinder::MatchResult &Result) {
         Repl += ";";
       }
       emplaceTransformation(new ReplaceText(Beg, Len, std::move(Repl)));
+      if (UD->getNameAsString() == "max" || UD->getNameAsString() == "min") {
```

Comment:
it should be better to remove using ::max instead of introduce dpct::max() function.

Reply:
ok